ProjFS.Mac/PrjFSKextTests/MemoryTests.mm

```diff
@@ -0,0 +1,34 @@
+//
+//  MemoryTests.m
+//  PrjFSKextTests
+//
+//  Created by John Briggs on 3/12/19.
+//  Copyright © 2019 Microsoft Corporation. All rights reserved.
```

Comment:
This automatic copyright should be removed (perhaps replaced with boilerplate MIT license comment?)

Reply:
Removed. I'll make a pass through at a later date to get correct headers on each file.